graphalyzer-server/src/file_handler/os/OSWrapper.py

```diff
@@ -49,7 +49,7 @@ def getOSPathHeader(self):
 			return "c:/"
 		else:
 			self.logger.debug("OS is Linux. Returning path header '/'")
-			return "/"
```

Comment:
The way it was before it was trying to write logs at /neo4j_file_handling which requires root permission. The way it works not is it puts it wherever you currently are. For example if you are at /home/yourname/somefolder/Graphalizer/graphalizer-server/ and run the server the logs will be there. This is better than what it was but we will probably need to change it later to allow a specific configurable location.
tl;dr leave it and fix it later.